cfme/v2v/migration_throttling.py

```diff
@@ -0,0 +1,61 @@
+import attr
+from navmazing import NavigateToAttribute
+from widgetastic.utils import WaitFillViewStrategy
+from widgetastic_patternfly import Button
+from widgetastic_patternfly import Input
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.modeling.base import BaseCollection
+from cfme.modeling.base import BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigator
+
+
+class MigrationSettingsView(BaseLoggedInPage):
+    fill_strategy = WaitFillViewStrategy("20s")
+    migration_per_conv_host = Input(id='max_concurrent_tasks_per_host')
+    migration_per_provider = Input(id='max_concurrent_tasks_per_ems')
+    apply_btn = Button("Apply")
+
+    @property
+    def is_displayed(self):
+        nav_menu = ["Compute", "Migration", "Migration Settings"]
+        return self.logged_in_as_current_user and self.navigation.currently_selected == nav_menu
+
+
+@attr.s
+class MigrationThrottling(BaseEntity):
+    """Class representing Migration throttling
+        Args:
+            migration_per_conv_host: maximum concurrent migration per conversion host
+            migration_per_provider: maximum concurrent migrtaion per provider
+    """
+    migration_per_conv_host = attr.ib(default=None)
+    migration_per_provider = attr.ib(default=None)
+
+
+@attr.s
+class MigrationThrottlingCollection(BaseCollection):
+    """Collection object for migration throttling"""
+
+    ENTITY = MigrationThrottling
+
+    def set_max_migration_per_conv_host(self, migration_per_conv_host):
+        """Migration throttling for conversion host
+        Args:
+            migration_per_conv_host: Integer
+        """
+        view = navigate_to(self, "All")
+        view.fill({'migration_per_conv_host': migration_per_conv_host})
+        view.apply_btn.click()
+        return self.instantiate(migration_per_conv_host=migration_per_conv_host)
+
+
+@navigator.register(MigrationThrottlingCollection, "All")
+class MigrationThrottling(CFMENavigateStep):
+    prerequisite = NavigateToAttribute("appliance.server", "LoggedIn")
+    VIEW = MigrationSettingsView
+
+    def step(self):
+        self.prerequisite_view.navigation.select("Compute", "Migration", "Migration Settings")
```

Comment:
Migration has it's own tab in the latest build of 5.11
![Screenshot from 2019-07-26 15-40-19](https://user-images.githubusercontent.com/44065123/61977188-c2980a80-afbb-11e9-9803-2ed01a14bfe7.png)